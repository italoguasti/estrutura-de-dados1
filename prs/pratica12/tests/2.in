50 milho fazendadaalegria
30 trigo sitiovaledapaz
70 soja fazendadaluz
20 cevada fazendaazul
50 cafe sitiocabecao
60 arroz fazendacross
80 algodao sitiogalo
0
10 70
