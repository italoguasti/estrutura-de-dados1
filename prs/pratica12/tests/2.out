InOrder:
20 ( [cevada|fazendaazul] )
30 ( [trigo|sitiovaledapaz] )
50 ( [milho|fazendadaalegria] [cafe|sitiocabecao] )
60 ( [arroz|fazendacross] )
70 ( [soja|fazendadaluz] )

PreOrder:
20 ( [cevada|fazendaazul] )
30 ( [trigo|sitiovaledapaz] )
60 ( [arroz|fazendacross] )
70 ( [soja|fazendadaluz] )
50 ( [milho|fazendadaalegria] [cafe|sitiocabecao] )

PosOrder:
50 ( [milho|fazendadaalegria] [cafe|sitiocabecao] )
30 ( [trigo|sitiovaledapaz] )
20 ( [cevada|fazendaazul] )
70 ( [soja|fazendadaluz] )
60 ( [arroz|fazendacross] )
